### PR TITLE
GEOMESA-2246 Delete READMEs in sub-modules

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Moved Jupyter and CQEngine information into docs

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: docs/user/spark/jupyter.rst [hunk 1]
```diff
@@ -119,13 +119,122 @@ GeoMesa version ``VERSION``:
...
+The following sample notebook shows how you can use Leaflet for data visualization:
+
+
+.. code-block:: scala
+
+   classpath.addRepository("http://download.osgeo.org/webdav/geotools")
+   classpath.addRepository("http://central.maven.org/maven2")
+   classpath.addRepository("https://repo.locationtech.org/content/repositories/geomesa-releases")
+   classpath.addRepository("file:///home/username/.m2/repository")
+   classpath.add("com.vividsolutions" % "jts" % "1.13")
```
Comment (jnh5y) [inline docs/user/spark/jupyter.rst]: Hmmm?  Seems like this block should be updated to jts-core version 1.14.0 and geomesa-* 2.0.*.

Comment (elahrvivaz) [reply docs/user/spark/jupyter.rst]: see, readme's are out of date ;)
i'll fix

**Comment by jahhulbert-ccri** [comment]
I am a :+1: on deleting them since there should be one canonical documentation with a better looking skin :)

**Comment by jahhulbert-ccri** [comment]
@jnh5y @aheyne I will click the button if you're ok with it.

**Pull request merged**